contrib/format-spss/README.md

```diff
@@ -0,0 +1,87 @@
+# Format Plugin for SPSS (SAV) Files
+This format plugin enables Apache Drill to read and query Statistical Package for the Social Sciences (SPSS) (or Statistical Product and Service Solutions) data files. According
+ to Wikipedia: [1]
+ 
+ SPSS is a widely used program for statistical analysis in social science. It is also used by market researchers, health researchers, survey companies, government, education researchers, marketing organizations, data miners, and others. The original SPSS manual (Nie, Bent & Hull, 1970) has been described as one of "sociology's most influential books" for allowing ordinary researchers to do their own statistical analysis. In addition to statistical analysis, data management (case selection, file reshaping, creating derived data) and data documentation (a metadata dictionary is stored in the datafile) are features of the base software.
```

Comment:
Blockquote may be used for quoting from Wiki.

Reply:
Fixed.